### DISCUSSION TITLE
XLIFF and CSV export for Optimizely CMS v12

### DISCUSSION THREAD

**Discussion by Cells2019** [valdisiljuconoks/localization-provider-epi#233] [General]
Any chance the XLIFF and CSV export packages will be updated to work with Optimizely CMS version 12?

**Comment by valdisiljuconoks** [comment]
hey, maybe it's not properly communicated from my side, but packages have been moved over to generic .net side of the fence and you can find them on nuget.

just search for provider and xliff or csv

**Comment by Cells2019** [comment]
Hi Valids, Thanks for getting back to me. I found the packages and they are installed. Does the Optimizely Admin UI package need to be updated for these options to show in the CMS Admin environment? Or is there something else I am missing. I currently only have the option to export from the Admin UI as JSON.

**Reply by valdisiljuconoks** [reply]
did you add these importers/exporters to the setup config?

**Reply by Cells2019** [reply]
I didn't realize I needed to do that. I will look and see if I can figure out how.

**Reply by valdisiljuconoks** [reply]
checking the docs I see that I'm missing documentation on how this is configurable. . should add more I guess

**Reply by Cells2019** [reply]
That would be great. I haven't figured it out yet.

**Reply by valdisiljuconoks** [reply]
it's just extension methods when adding localization provider to your service collection..

```
services
    .AddDbLocalizationProviderAdminUI(ctx => { .... })
    .AddOptimizelyAdminUI()
    .AddCsvSupport()
    .AddXliffSupport();
```

**Reply by Cells2019** [reply]
I got it to work. I was missing the AdminUI packages for Xliff and CSV as well. All good now. Thank you for your help.
